external-api/data-types.md

```diff
@@ -834,6 +834,13 @@ A booking can belong to an order, but it doesn't have to. An order can include a
 
   When there is no order associated with the booking, `order` will be `null`.
 
+* `dashboard_url`: `string`
+  The booking's dashboard URL.
+
+* `customer_count`: `number`
+  The number of customers on this booking
```

Comment:
To be consistent, we should have a period at the end of this line.

Reply:
updated